doc/topics/releases/nitrogen.rst

```diff
@@ -61,48 +61,50 @@ State Module Changes
   start/stop the service using the ``--no-block`` flag in the ``systemctl``
   command. On non-systemd minions, a warning will be issued.
 
-- The :py:func:`module.run <salt.states.module.run>` state has dropped its previous
-  syntax with ``m_`` prefix for reserved keywords. Additionally, it allows
-  running several functions in a batch.
+- The :py:func:`module.run <salt.states.module.run>` state has dropped its
+  previous syntax with ``m_`` prefix for reserved keywords. Additionally, it
+  allows running several functions in a batch.
 
-.. note::
-    It is nesessary to explicitly turn on the new behaviour (see below)
+  .. note::
+      It is necessary to explicitly turn on the new behavior (see below)
 
-  Before and after:
+  .. code-block:: yaml
 
-.. code-block:: yaml
+      # Before
+      run_something:
+        module.run:
+          - name: mymodule.something
+          - m_name: 'some name'
+          - kwargs: {
```

Comment:
I realize this part wasn't really changed, but since it's green in the PR... Do braces really work like this in yaml? I'd think, at the least, the keys would need to be quoted, no? Maybe just kill the braces and indent the lines two more spaces so it's recognized as a yaml nested dict?